src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java

```diff
@@ -177,8 +178,7 @@ private void addReportSummary(HearingsToJudgmentsReportSummary reportSummary,
                 hearingJudgmentItem.judgmentDateSent = dateJudgmentSent.format(OLD_DATE_TIME_PATTERN2);
                 hearingJudgmentItem.total = hearingListedDate.datesUntil(dateJudgmentSent).count();
                 hearingJudgmentItem.reservedHearing = dateListedType.getHearingReservedJudgement();
-                hearingJudgmentItem.judge = hearingType.getJudge();
-
+                hearingJudgmentItem.judge = hearingType.getJudge().substring(hearingType.getJudge().indexOf('_') + 1);
```

Comment:
May be this looks a bit cleaner?
Strings.split(hearingType.getJudge(), '_')[1]

Reply:
Yh, may be you can leave it as it was or use this :) 

String[] segments = Strings.split(hearingType.getJudge(), "_");
                hearingJudgmentItem.judge = segments[segments.length - 1];

Reply:
Will see what other reviewers say

Reply:
I think checking for null is the simplest and more readable way.

Reply:
Added a null check and small refactor